MAUI/ImageEditor/shape.md

```diff
@@ -51,6 +51,84 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Polygon
```

Comment:
Images are not clear, add shapes without background image and try to add large size shapes. For polyline add shape like in whatsnew image